xcube/util/cmaps.py

```diff
@@ -402,6 +412,17 @@ def get_cmap(
         return cmap, colormap
 
     def to_json(self) -> list:
+
+        def get_cmap_info(cm_name):
+            if self._colormaps[cm_name].cm_code is not None:
+                return [
+                    cm_name,
+                    self._colormaps[cm_name].cmap_png_base64,
+                    self._colormaps[cm_name].cm_code,
```

Comment:
I prefer putting a JSON object here rather than a string.

Reply:
Done

Reply:
Where?

Reply:
Here: 

https://github.com/xcube-dev/xcube/pull/1055/files#diff-90ed9178e6fa0059e53b3b5b7aab7826e770ba5c6b6be8b5e64fda597ea6c91eR539

Instead of using `cm_code` I add the JSON object and add to the colorbar endpoint later.